### PR TITLE
Update functional tests for Protocol pages (Fixes #5995)

### LINKED ISSUE #5995: Write functional tests for recently created Protcol pages
The new Protocol newsletter component is on the Home and About pages now. We should write new functional tests to monitor the functionality like we did the old forms on those pages.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds new functional tests for recently created Protcol pages.

## Issue / Bugzilla link
#5995

## Testing
Successful test run in Sauce Labs: https://ci.us-west.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/213/pipeline/

### CODE REVIEW

**Comment by alexgibson** [comment]
@jpetto I would normally ask Paul here, but seeing as he's out this week I thought you might have some time for a Python review? :)

File: tests/functional/firefox/test_pocket.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import pytest
+
+from pages.firefox.pocket import FirefoxPocketPage
+
+
+@pytest.mark.nondestructive
+def test_pocket_learn_more_button(base_url, selenium):
+    page = FirefoxPocketPage(selenium, base_url).open()
+    assert page.is_pocket_primary_button_displayed
+    assert page.is_pocket_primary_button_displayed
```
Comment (jpetto) [inline tests/functional/firefox/test_pocket.py]: This should probably be `is_pocket_secondary_button_displayed`? 🍝

Comment (alexgibson) [reply tests/functional/firefox/test_pocket.py]: 🤦‍♂️

**Comment by alexgibson** [comment]
Thanks @jpetto - updated!

**Pull request merged**